bin/git-test

```diff
@@ -468,7 +468,8 @@ class Test(object):
 
         try:
             process = subprocess.Popen(cmd, **kwargs)
-            out, err = communicate(process, ''.join('%s^{tree}\n' % (r,) for r in revisions))
+            rev_args = ''.join('%s^{tree}\n' % (r,) for r in revisions)
```

Comment:
I extracted this `rev_args` variable since I felt the original line was too long and busy to just casually "hide" a `[0]` at the end.